core_planning/mpc_follower/launch/mpc_follower.launch

```diff
@@ -37,18 +39,29 @@
 
 	<group ns="$(arg ns)">
 
-		<node pkg="mpc_follower" type="mpc_waypoints_converter" name="mpc_waypoints_converter" output="screen"/>
+		<node pkg="mpc_follower" type="mpc_waypoints_converter" name="mpc_waypoints_converter" output="screen">
+			<remap from="/base_waypoints" to="/guidance/base_waypoints"/>
+			<remap from="/closest_waypoint" to="/guidance/closest_waypoint"/>
+			<remap from="/final_waypoints" to="/guidance/carma_final_waypoints"/>
+		</node>
 
 		<node pkg="mpc_follower" type="mpc_follower" name="mpc_follower" output="screen">
 
+			<remap from="/ctrl_raw" to="/guidance/ctrl_raw"/>
+			<remap from="/twist_raw" to="/guidance/twist_raw"/>
+			<remap from="/vehicle_status" to="/hardware_interface/vehicle_status"/>
+			<remap from="/current_pose" to="/localization/ndt_pose"/>
```

Comment:
Same as above. All the /guidance namespaces are added automatically at runtime so /ctrl_raw to ctrl_raw should be sufficient. The /hardware_interface and /localization remappings should be done in guidance.launch but in addition current_pose is the correct topic not ndt_pose.